### PR TITLE
MESH-823/schema test script

### PR DESCRIPTION
**Pull request by JMoore96**

### CODE REVIEW

File: scripts/cli/util/schema_check.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+let { reqImports } = require("./init.js");
+
+async function main() {
+  // Schema path
+  const filePath = reqImports["path"].join(__dirname + "/../../../polymesh_schema.json");
+  const customTypes = JSON.parse(reqImports["fs"].readFileSync(filePath, "utf8"));
+
+  // Start node instance
+  const ws_provider = new reqImports["WsProvider"]("ws://127.0.0.1:9944/");
+  const api = await reqImports["ApiPromise"].create({
```
Comment (vkomenda) [inline scripts/cli/util/schema_check.js]: Variable declaration isn't needed here, so you could remove `const api =`.

File: scripts/cli/util/schema_check.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    types: customTypes,
+    provider: ws_provider
+  });
+
+  process.exit();
```
Comment (vkomenda) [inline scripts/cli/util/schema_check.js]: Why is this needed? Doesn't the process terminate after `main` finishes? I think `process.exit()` can be removed unless it helps to fix something else.

Comment (JMoore96) [reply scripts/cli/util/schema_check.js]: You are correct, I will make the changes you suggested thanks.

File: scripts/cli/util/schema_check.js (hunk 1 above)
Comment (JMoore96) [reply scripts/cli/util/schema_check.js]: alright, will do.
